src/app/translations/en.ts

```diff
@@ -55,7 +55,7 @@ export const translation: ITranslations = {
                 generic:
                     'Sorry, there was an error posting your transaction to blockchain. Try again or come back later.'
             },
-
+            addAllBalance: 'ADD ALL BALANCE',
```

Comment:
buttons already have upercase transformation from css you can use here `Add all balance`

Reply:
Done